pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -317,11 +317,11 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 	case metal3v1alpha1.StateDeprovisioning:
 		result, err = r.actionDeprovisioning(prov, info)
 	case metal3v1alpha1.StateProvisioned:
-		result, err = r.actionManageHostPower(prov, info)
+		result, err = r.actionManageSteadyState(prov, info)
 	case metal3v1alpha1.StateReady:
-		result, err = r.actionManageHostPower(prov, info)
+		result, err = r.actionManageSteadyState(prov, info)
```

Comment:
Ready hosts aren't provisioned, if we adopt them in Ironic then Ironic will think they are.

Reply:
Fixed.